src/mqtt.cpp

```diff
@@ -505,6 +507,23 @@ mqtt_publish_schedule() {
   }
 }
 
+bool
+mqtt_publish_config() {
+  if(!config_mqtt_enabled() || !mqttclient.connected() || evse.getEvseState() == OPENEVSE_STATE_STARTING) {
+    return false;
+  }
+  const size_t capacity = JSON_OBJECT_SIZE(128) + 1024;
+  DynamicJsonDocument doc(capacity);
+  config_serialize(doc, true, false, true);
+  mqtt_publish_json(doc, "/config");
+
+  String fulltopic = mqtt_topic + "/config_version";
+  String payload = String(config_version());
+  mqttclient.publish(fulltopic, payload, true);
```

Comment:
I don't think you need to event `config_version`, this is already evented via event_send when the version is incremented.